test/components/connect.spec.js

```diff
@@ -203,8 +202,8 @@ describe('React', () => {
 
       const pending = TestUtils.findRenderedComponentWithType(container, Container)
       const startedAt = pending.state.startedAts.testFetch
-      expect(startedAt.getTime()).toBeLessThan(new Date().getTime())
```

Comment:
I'm not sure why but when run in watch mode this test was failing because these two times were equal